packages/ovee/src/errors/Logger.ts

```diff
@@ -0,0 +1,73 @@
+import { isString } from 'src/utils/isString';
+
+type LogLevel = 'log' | 'warn' | 'info' | 'error';
+
+export class Logger {
+	constructor(public subnamespace = '', public namespace = 'Ovee') {}
```

Comment:
maybe we move the framework name into a centralized constant, so if we change a convention e.g. to smallcaps or using `.js` suffix, we don't miss any occurance

Reply:
added constant `FRAMEWORK_NAME = 'Ovee.js'`